.github/workflows/auto-merge-bot.yml

```
@@ -0,0 +1,19 @@
+on: [pull_request]
+
+jobs:
+  auto_merge_bot:
+    runs-on: ubuntu-latest
+    name: EIP Auto-Merge Bot
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Node.js Environment
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - name: auto-merge-bot
+        uses: alita-moore/EIP-Bot@1.1.1 # v1.1.1 does not have the ability to merge and throws and error on non-merge like posting comment
```

```suggestion
        uses: alita-moore/EIP-Bot@83def55233002790da6746afabe4724b8ee4f940 # tag 1.1.1
```
Does this work?  From a security standpoint, it protects us from a number of attack vectors including compromised keys, etc.  Not a critical thing, but if it works I have a slight preference for it.

---

Oh yeah this would work! Cool standpoint though about security. The more you know 😀